config/test.yaml

```diff
@@ -1,5 +1,6 @@
 asyncRequestApi: {
-    url: http://localhost:8082
+    url: http://localhost:8080,
+    port: 8080
```

Comment:
Do you use the `port` or just rely on it being present in the `url`?  If the latter, maybe just rename `url` to `baseUrl` and then get rid of `port`?

Reply:
I use the port when setting up the wiremock test container, and tried changing url to baseUrl without the port. but the issue is I'd have to then append port to base url in the code and for none local URLs I'd end up with the port being appended. I.e. https://publish-api.planning.data.gov.uk:8082